x/bank/keeper/keeper.go

```diff
@@ -24,8 +24,12 @@ type Keeper interface {
 	InitGenesis(sdk.Context, *types.GenesisState)
 	ExportGenesis(sdk.Context) *types.GenesisState
 
-	GetSupply(ctx sdk.Context) exported.SupplyI
-	SetSupply(ctx sdk.Context, supply exported.SupplyI)
+	GetSupply(ctx sdk.Context, denom string) sdk.Coin
+	GetTotalSupply(ctx sdk.Context) sdk.Coins
+	SetSupply(ctx sdk.Context, supply sdk.Coins)
+	InflateSupply()
+	DeflateSupply()
```

Comment:
I think we're wanting to remove all of this `SetSupply` type stuff in line with #8416 right?

Reply:
The plan is to let the SetSupply stay and clean it up in the bank refactor, rather than addressing two issues together. I'm getting rid of the Inflate and Deflate methods though.